server/features/EmailInbox/EmailInbox_Incoming.ts

```diff
@@ -30,12 +31,20 @@ const language = settings.get('Language') || 'en';
 const t = (s: string): string => TAPi18n.__(s, { lng: language });
 
 function getGuestByEmail(email: string, name: string, department?: string): any {
+	logger.debug(`Attempt to register a guest for ${ email } on department: ${ department }`);
 	const guest = LivechatVisitors.findOneGuestByEmailAddress(email);
 
 	if (guest) {
+		logger.debug(`Guest with email ${ email } found with id ${ guest._id }`);
+		if (guest.department !== department) {
+			logger.debug(`Switching departments for guest ${ guest._id }. Previous: ${ guest.department } New: ${ department }`);
```

Comment:
for debugging purposes might be useful to add extra data, for this it is useful logging big objects with an additional message, like:
```js
logger.debug({
  msg: 'Switching departments for guest',
  guest,
  new: department,
});
```